metadata.yaml

```diff
@@ -7,12 +7,68 @@ description: |
 requires:
```

Comment:
@DnPlas Do we want to add these at this point?
```
website: https://charmhub.io/dex-auth
source: https://github.com/canonical/dex-auth-operator
issues: https://github.com/canonical/dex-auth-operator/issues
docs: https://discourse.charmhub.io/t/charmed-dex/8214
```

Reply:
Otherwise LGTM

Reply:
@i-chvets let's add it in a follow-up PR so both main and track/2.31 have the change.

Reply:
+1